handlers/admin/recompute_sm5_scores.py

```diff
@@ -0,0 +1,44 @@
+from sanic import Request
+
+from db.sm5 import SM5Game, SM5_LASERRANK_VERSION
+from helpers.sm5helper import update_winner
+from shared import app
+from utils import admin_only
+
+_BATCH_SIZE = 20
+
+
+@app.get("/admin/recompute_sm5_scores")
+@admin_only
+async def recompute_sm5_scores(request: Request) -> str:
+    response = await request.respond(content_type="text/html")
+
+    await response.send("<html><body><H1>Updating...</H1>\n")
```

Comment:
uncapitalize h1 (is that html/body tag required?)

Reply:
Done.
I'm looking at the raw source coming from the server and it doesn't have duplicate HTML and BODY, so it does seem to be required. This is raw output and not using any templates.